txnsync/bitmask.go

```diff
@@ -113,27 +113,33 @@ func (b *bitmask) trimBitmask(entries int) {
 }
 
 // iterate through the elements of bitmask without expanding it.
-// If more than maxIndex set bits are found, return error. For each set value, call callback.
-func (b *bitmask) iterate(entries int, maxIndex int, callback func(int, int) error) error {
+// call the func(entriesCount, setBitIndex) for every set bit
+// countOfEntries: is the size of the array that entryIndex is accessing: entryIndex < countOfEntries
+// countOfSetBits: is the size of the array that setBitCounter is accessing: setBitCounter < countOfSetBits
+func (b *bitmask) iterate(countOfEntries int, countOfSetBits int, callback func(int, int) error) error {
```

Comment:
I suggest countOfEntries -> numTransactions, since this number refers to the number of transactions that are present.

I suggest countOfSetBits -> numItems, since this refers to the number of items we are iterating over.

Reply:
then correspondingly, entryIndex -> transactionIndex and setBitCounter -> itemIndex

Reply:
@nicholasguoalgorand, is it always the case the the entries count are the transactions count ?

Reply:
yes

Reply:
In that case I agree - numTransactions or transactionCount would make it more readable.